tests/system/python/conftest.py

```diff
@@ -245,7 +250,7 @@ def _start_north_omf_as_a_service(fledge_url, pi_host, pi_port, pi_db="Dianomic"
 
 
 start_north_pi_server_c = start_north_pi_v2
-start_north_pi_server_c_web_api = start_north_pi_v2_web_api
```

Comment:
I would suggest to keep 

`# As we had name pi_server, pi_v2 in old versions for OMF PI server things
start_north_pi_server_c_web_api = start_north_pi_v2_web_api = start_north_task_omf_web_api